### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @SolohagaLukas

### CODE REVIEW

**Comment by SolohagaLukas** [comment]
Profe terminé

File: README.md [hunk 1]
```diff
@@ -23,3 +23,7 @@ Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archi
 ![Directamente desde un link](http://www.plantuml.com/plantuml/png/JSqn2W9138NXFgT8gLUmMb5hlS9Z9gu3cGmaMH74kxl25lR_zzuK2S-bAnL51DyKC_XDpEcmw2tbOdVnBXLrjp_mKUT17zhoqrBWA0y4ool5wl-2jQVe6TR63rxJnuQdDLmllm00)
 
 ![Desde un archivo local](./images/ejemplo-uml.png)
+
+Link de Barrio (http://www.plantuml.com/plantuml/uml/bOin2i9044NxFSMKIU45B2Ii26dY7RwR8GYpEpApMndlRa05mShkyzv_WrTORLb24hXJXDbLwHw8i1Wli9DmvbAru_VOTwDkV1qRMyKwmPDMzFsMED9_mUVYiN5HxthQ7UZSyeNjsqwGN-b7lM7M5uwgmYZX4GOkSylo1000)
```
Comment (faloi) [inline README.md:27]: Te faltaron unos corchetes y un ! para que te funcione:

```suggestion
![Link de Barrio](http://www.plantuml.com/plantuml/uml/bOin2i9044NxFSMKIU45B2Ii26dY7RwR8GYpEpApMndlRa05mShkyzv_WrTORLb24hXJXDbLwHw8i1Wli9DmvbAru_VOTwDkV1qRMyKwmPDMzFsMED9_mUVYiN5HxthQ7UZSyeNjsqwGN-b7lM7M5uwgmYZX4GOkSylo1000)
```

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,21 +1,26 @@
 class Barrio {
 
 	const property elementos = []
```
Comment (faloi) [inline src/barrio.wlk:3]: Falta el tipo.

Comment (faloi) [inline src/barrio.wlk:3]: Falta el tipo.

**Pull request closed without merging**